Makefile

```diff
@@ -48,7 +44,6 @@ smoke-test:
 	BCDA_SSAS_CLIENT_ID=$(SSAS_ADMIN_CLIENT_ID) BCDA_SSAS_SECRET=$(SSAS_ADMIN_CLIENT_SECRET) CLIENT_ID=$(CLIENT_ID) CLIENT_SECRET=$(CLIENT_SECRET) docker-compose -f docker-compose.test.yml run --rm -w /go/src/github.com/CMSgov/bcda-app/test/smoke_test tests sh smoke_test.sh
 
 smoke-test-ssas:
```

Comment:
Do we still need this?

Reply:
Yes, it runs the ACO API->SSAS integration test we've talked about

Reply:
There is definitely some redundancy here, that could potentially be cleaned up.  Here are some thoughts/concerns:
(1) The `smoke-test` target ultimately runs `smoke_test.sh`.  But, the `smoke-test-ssas` target _also_ runs `smoke_test.sh` (after doing some initial SSAS-related setup).  Could we remove the `smoke-test-ssas` target, rename `ssas_test.sh` to `smoke_test.sh` (overwriting the current `smoke_test.sh`) since `ssas_test.sh`  is simply doing BCDA-related CLI calls with SSAS as the provider, and just keep one `smoke-test` Makefile target?
(2) Could we configure the `docker-compose` files so that SSAS is the default provider at all times, now that we've set up the stack with SSAS running?